modules/gdscript/gdscript_editor.cpp

```diff
@@ -381,9 +381,11 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 }
 
 String GDScriptLanguage::debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems, int p_max_depth) {
-
-	if (_debug_parse_err_line >= 0)
+	//TODO
+	/*
+	 if (_debug_parse_err_line >= 0)
 		return "";
+	*/
```

Comment:
CC @vnen @bojidar-bg
What was the intent of this method?

Reply:
It implements something similar to `gdb`'s `print`/`p` command, by evaluating an expression and printing its result. It isn't used outside the command-line debugger, and doesn't seem to have been implemented either.